Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,285 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling;
+model DirectControlled "Direct cooling ETS model for district energy systems with in-building 
```

Comment:
Thanks for the question! The "in-building pumping" term is from the first implementation by Katy, and I checked the reference where "in-building equipment" is used in the District Cooling Guide.

The `deltaT` is referred to the temperature difference between supply water temperature and return water temperature. Since the supply temperature is fixed, only return temperature can be controlled. But you made a really good point, I will use `deltaT control` for both places.